apps/SageAccountWeb/src/components/StyledComponents.ts

```diff
@@ -31,8 +31,7 @@ export const StyledInnerContainer: StyledComponent<PaperProps> = styled(Paper, {
   width: '900px',
   margin: '0 auto',
   display: 'flex',
-  boxShadow: '0px 2px 6px rgba(53, 58, 63, 0.1)',
-  borderRadius: '5px',
+  overflow: 'hidden',
```

Comment:
Fix border-radius on StyledInnerContainer and use Paper styling from theme overrides

main | feature
:---:|:---:
![main_styledcontainer](https://user-images.githubusercontent.com/26949006/229907380-b1317ef8-67f5-46f0-a89f-5be054addc4f.png) | ![feature_styledcontainer](https://user-images.githubusercontent.com/26949006/229907372-eb4e953b-ffd5-4f37-8570-cba629fad99b.png)